src/BaseAssets/FileSystem/Common.mo

```diff
@@ -47,7 +41,7 @@ module {
     };
 
     public func get_html_alias_root(fs : T.FileSystem, alias : Text) : ?Text {
-        if (not Text.endsWith(alias, #text ".html")) return null;
```

Comment:
@tomijaga This an a few other changes were added to TRY to make it so that when i add an asset called `index.html`, i would be able to get a certified form from `/`, but instead i would always get the index.html, but certified for the /index.html path, vs the / path
I don't think i was successful, and this is my biggest issue right now
From what i could figure out, if you added a file as /, it would resolve to /index.html, but not the other way around (i could be wrong, but thats what it looks like)